service_mesh/v2x/ossm-gateway-migration.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-gateway-migration"]
+= Gateway Migration
+include::_attributes/common-attributes.adoc[]
+:context: gateway-migration
+
+toc::[]
+
+As a network administrator, the preferred method for deploying ingress and egress gateways is with a `Deployment` resource using gateway injection.
+
+include::modules/ossm-about-gateway-migration.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-from-smcp-defined-gateways-to-gateway-injection.adoc[leveloffset=+1]
+
+[id="additional-resources_{context}"]
+== Additional resources
```

Comment:
This part still needs the role attribute per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections):
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```

Reply:
done